cookietemple/create/template_creator.py

```diff
@@ -361,8 +361,8 @@ def create_dot_cookietemple(self, template_version: str):
         :param template_version: Version of the specific template
         """
         log.debug('Creating .cookietemple.yml file.')
-        self.creator_ctx.template_version = f'{template_version} # <<COOKIETEMPLE_NO_BUMP>>'
-        self.creator_ctx.cookietemple_version = f'{cookietemple.__version__} # <<COOKIETEMPLE_NO_BUMP>>'
+        self.creator_ctx.template_version = f"{template_version} # <<COOKIETEMPLE_NO_BUMP>>"
+        self.creator_ctx.cookietemple_version = f"{cookietemple.__version__} # <<COOKIETEMPLE_NO_BUMP>>"
```

Comment:
why `"` instead of `'`?

Reply:
That was to satisfy prettier. We are now ignoring .cookietemple.yml -> I will revert this. Good catch.